packages/commonwealth/package.json

```diff
@@ -84,7 +85,8 @@
     "@cosmjs/amino": "^0.31.3",
     "@cosmjs/cosmwasm-stargate": "^0.31.3",
     "@cosmjs/crypto": "0.31.3",
-    "@cosmjs/encoding": "0.32.3",
+    "@cosmjs/encoding": "0.31.3",
```

Comment:
Downgrading here is causing a build error -> https://github.com/cosmos/cosmjs/issues/1573

Reply:
That may have been accidental...Have reverted and waiting to see if it passes